.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Do we need this?

Reply:
We have it in some other repo, like helix-cli. It is quite useful to not have to re-create it each time, especially the "Mocha Current File". I do not think we have a rule yet, it is IDE specific thus both having it or not could be argued. I am fine removing it.